lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -105,7 +108,7 @@ define([
                 staff: false,
                 team_memberships_data: createMockTeamMembershipsData(1, 5)
             },
-            options
+            options || {}
```

Comment:
`_.extend` treats non-object parameters as empty objects, so this change isn't necessary.